Source/WebCore/PAL/pal/text/TextEncoding.h

```diff
@@ -37,6 +37,7 @@ class TextEncoding : public WTF::URLTextEncoding {
 public:
     TextEncoding() = default;
     PAL_EXPORT TextEncoding(const char* name);
```

Comment:
Do we still need this one?

Reply:
I will double check but as I remember it, there were still call sites that required it.

Reply:
Actually, I can remove this with minor amount of work.